test/osmt-apitest.env.example

```diff
@@ -0,0 +1,4 @@
+OKTA_URL=https://dev-XXXXXXXX.okta.com
+LOCAL_URL=http://localhost:8080
+OKTA_USERNAME=XXX.XXX@example.com
+OKTA_PASSWORD=XXX
```

Comment:
I had problems with my password, because contained `$`, solution here was use `\$` but `https://dev-XXXXX.okta.com/api/v1/authn` returned 401, so I think was using `\` as part of the password.  
Do we need to consider that a password can contain an escape character?